scalaz/src/main/scala/github4s/scalaz/Implicits.scala

```diff
@@ -0,0 +1,25 @@
+package github4s.scalaz
+
+import cats.MonadError
+import github4s.free.interpreters.Interpreters
+import scalaz.concurrent.Task
+
+object Implicits {
+
+  implicit val evalMonadError: MonadError[Task, Throwable] = new MonadError[Task, Throwable] {
```

Comment:
The name should be `taskMonadError`. Note that implicits can clash by name, so maybe we want to uniquify the names of the implicits throughout the library. In this case we could use `github4sTaskMonadError` for example.